src/goGraph/node-templates/symbol-node-template.ts

```diff
@@ -28,25 +130,95 @@ export function createSymbolNodeTemplate(clickHandler?: ((e: go.InputEvent, this
 				// more ContextMenuButtons would go here
 			), // end Adornment
 		})
-			.bind('width', 'symbolWidth', (w) => w + nodePadding * 2)
-			.bind('height', 'symbolHeight', (h) => h + nodePadding * 2)
-			// SYMBOL
+			// MARGIN PANEL
+			.add(
+				new go.Panel(go.Panel.Spot, {
+					background: colorizeLayers ? 'orange' : 'transparent',
+					alignment: go.Spot.Center,
+				})
+					.bind('width', 'symbolWidth', (w) => w + nodePadding * 2)
+					.bind('height', 'symbolHeight', (h) => h + nodePadding * 2)
+			)
+
+			// TOP LABEL
+			.add(
+				new go.Panel(go.Panel.Auto, {
```

Comment:
Can this panel go into separate file?

Reply:
Yes. I will tidy up this file in a later PR.

Reply:
👍